### ISSUE TITLE
[stable/grafana] ResourceQuotas when persistence is enabled

### ISSUE THREAD

**Issue by Jdavid7x** [helm/charts#12395]
Helm v2.11.0

Grafana chart/stable

We need to apply resource quotas in the initContainers when the persistence and the initchowndata is enabled. 

I will create a pull requests with the suggested changes.

You can create a namespace with resourcequota and install the chart with the below values to reproduce:

```
grafana:
  resources:
    limits:
      memory: 256Mi
      cpu: 250m
    requests:
      memory: 256Mi
      cpu: 250m
  rbac:
    create: false
    pspEnabled: false
  persistence:
    enabled: true
    initChownData: true
    existingClaim: grafana
  service:
    type: NodePort
```

**Comment by maorfr** [comment]
can this issue be closed?

**Comment by Jdavid7x** [comment]
Yes. I implemented the new version in our environment and is working. 

Thanks

**Comment by maorfr** [comment]
go ahead and close then ;)

**Comment by Jdavid7x** [comment]
Fixed on the version 3.0.0
